### PR TITLE
Add install-create event handlers

### LINKED ISSUE #532: Update and test automatic ticket creation
It's super old lol, probably needs to be fixed 

https://github.com/Andrew-Dickinson/join-form-request-bot

### PR DESCRIPTION
**Pull request by Andrew-Dickinson**
Brings the logic for creating slack messages and OSTicket tickets, which was contained in a separate service, into the meshdb monolith

Also fix an issue with admin UI redirects

Fixes #532

### CODE REVIEW

**Comment by Andrew-Dickinson** [comment]
Todo:

- [ ] Add env variables to IAC
- [ ] Tests?

File: src/meshapi/util/events/osticket_creation.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+import requests
+from django.db.models.base import ModelBase
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from meshapi.models import Install
+
+OSTICKET_API_TOKEN = os.environ.get("OSTICKET_API_TOKEN")
+OSTICKET_NEW_TICKET_ENDPOINT = os.environ.get("OSTICKET_NEW_TICKET_ENDPOINT")
```
Comment (james-otten) [inline src/meshapi/util/events/osticket_creation.py:13]:
```suggestion
OS_TICKET_URL = os.environ.get("OSTICKET_URL", "https://support.nycmesh.net")
OSTICKET_NEW_TICKET_ENDPOINT = f"{OS_TICKET_URL}/api/http.php/tickets.json"
```

File: .env.sample [hunk 2]
```diff
@@ -65,5 +65,8 @@ PG_ADMIN_URL=/pgadmin/
 FORMS_URL=https://devforms.mesh.nycmesh.net
 
 OSTICKET_URL=https://support.nycmesh.net
+OSTICKET_API_TOKEN=
+OSTICKET_NEW_TICKET_ENDPOINT=https://devsupport.nycmesh.net/api/http.php/tickets.json
```
Comment (james-otten) [inline .env.sample:69]:
```suggestion
```

File: src/meshapi/util/events/osticket_creation.py (hunk 1 above)
Comment (Andrew-Dickinson) [reply src/meshapi/util/events/osticket_creation.py:13]: Hmm the one drawback to this approach is that we won't be able to link to prod tickets from dev MeshDB, without _also_ cutting new tickets from installs created in dev to prod (super bad, the synthetics tests make new installs every 5 minutes)

Comment (james-otten) [reply src/meshapi/util/events/osticket_creation.py:14]: I see, good call

**Comment by Andrew-Dickinson** [comment]
Moved redirect fix to a standalone PR: #637

**Review by WillNilges — APPROVED** [review]
i hate osticket so much

**Pull request merged**